spec/clipboard.spec.js

```diff
@@ -34,6 +96,39 @@ describe('clipboard.spec: clipboard namespace tests', () => {
             `);
             assert.equal(runner.getOutput(), 'Test value');
         });
+        it('returns an empty string if the clipboard is empty', async () => {
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.clear();
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
+            `);
+            assert.equal(runner.getOutput(), '');
+        });
+        it('returns the same text for multiple consecutive reads', async () => {
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.writeText('Test value');
+                let clipboardText1 = await Neutralino.clipboard.readText();
+                let clipboardText2 = await Neutralino.clipboard.readText();
+                await __close(clipboardText1 === clipboardText2 ? 'ok' : 'not ok');
```

Comment:
Do you think it's better to do the comparison from Node with `assert` by sending both strings via an object?